src/LocGridOrbitals.h

```diff
@@ -95,7 +95,7 @@ class LocGridOrbitals : public Orbitals
     ////////////////////////////////////////////////////////
     // instance specific data
     ////////////////////////////////////////////////////////
-    BlockVector<ORBDTYPE, MemorySpace::Host> block_vector_;
+    BlockVector<ORBDTYPE, memory_space_type> block_vector_;
```

Comment:
I think we should keep that object on the host for now, and not have GPU support for LocGridOrbitals.
Let's focus on having only the alternate class 'ExtendedGridOrbitals' on the GPU for now. (we use either one depending on the run time option chosen)